lib/models/users.js

```diff
@@ -0,0 +1,16 @@
+const database = require("../../config");
+
+class User {
+  constructor(obj) {
+    this.email = obj.email,
+    this.password = obj.password,
+    this.apiKey = obj.apiKey,
+    this.favoriteLocation = obj.favoriteLocation;
+  }
+
+  static find(key) {
+    return database("users").where({ apiKey: key });
```

Comment:
YES! i had to go backwards and do this. smart move!!!

Reply:
Thanks! But I'm still not sure if this function returns a boolean (ie. whether this user exists)?